package_control/downloaders/curl_downloader.py

```diff
@@ -142,8 +141,8 @@ def download(self, url, error_message, timeout, tries, prefer_cached=False):
             try:
                 output = self.execute(command)
 
-                with open_compat(self.tmp_file, 'r') as f:
-                    headers_str = read_compat(f)
+                with open(self.tmp_file, 'r', encoding='utf-8', errors='replace') as fobj:
```

Comment:
I looked and could not find any reference to curl explicitly encoding headers as UTF-8. HTTP should be using ISO-8859-1, so we should use that when reading from the headers.

Reply:
utf-8 is what read_compat used to use. Changing to ISO-8859-1 shouldn't make an difference in case headers use ascii charset only.

How about reading from stderr then? It appears everything is decoded as utf-8 explicitly. May it be safe to just omit `encoding` argument to let python choose the best encoding for each OS?

Reply:
... see line 165 for instance.